test/conftest.py

```diff
@@ -1,35 +1,166 @@
 from __future__ import absolute_import
 
-import os
-
+import inspect
 import pytest
+import time
 
-from test.fixtures import KafkaFixture, ZookeeperFixture
+from decorator import decorator, decorate
 
+from kafka import MultiProcessConsumer, SimpleConsumer
+from test.fixtures import KafkaFixture, ZookeeperFixture, get_simple_consumer, get_simple_producer, get_keyed_producer, get_base_producer, get_simple_client
+from test.testutil import kafka_version, version_str_to_list, random_string, wait_for_kafka_client_topic_update, wait_for_simple_client_topic_update
 
 @pytest.fixture(scope="module")
 def version():
-    if 'KAFKA_VERSION' not in os.environ:
-        return ()
-    return tuple(map(int, os.environ['KAFKA_VERSION'].split('.')))
-
+    return kafka_version()
 
 @pytest.fixture(scope="module")
-def zookeeper(version, request):
-    assert version
```

Comment:
I wonder why these were in here in the first place...?

Reply:
I can't tell. But having the check on the Kafka fixture seems enough to me... ZK shouldn't depend on that for anything.